README-windows.md

```diff
@@ -19,21 +19,16 @@
     1. Open a command prompt and navigate to the directory with the vendor-sdk repo.
     2. Run the following commands:
 
-            mkdir build
-            cd build
-            cmake -G "MSYS Makefiles" ..
-            make
+            cmake -S . -B build -G "MSYS Makefiles" ..
+            cmake --build build
```

Comment:
Good addition, the original commands were using an older version of cmake and building with `make`.
The newer version allows for autocreation of the folder.